### ISSUE TITLE
So You Wanna Be a Pandas Expert? (Tutorial) - James Powell | PyData Global 2021

### ISSUE THREAD

**Issue by torres-federico-e** [numfocus/YouTubeVideoTimestamps#70]
[So You Wanna Be a Pandas Expert? (Tutorial) - James Powell | PyData Global 2021](https://www.youtube.com/watch?v=pjq3QOxl9Ok)

Claim on Pandas Expertise video, by @James Powell

Will be uploading Timestamps on few moments

**Comment by Alt-Shivam** [comment]
Hey @torres-federico-e 
Can you help me out to understand this repo 
I mean there are lots of issues open 
but still, no PRs to resolve them 
I am just kind of confused.
Thanks.

**Comment by torres-federico-e** [comment]
Of course. The PyData YouTube Channel has opened for collaboration all the Time Stamping efforts for indexing videos of their conferences

 You basically claim video from YouTube that you want to watch,  watch it, make the timestamps, and create an issue on this repo and collaborate.

They will add them to YouTube, make a PR, and credit for your help to the community afterwards.

**Comment by Alt-Shivam** [comment]
Got it.
Thanks a lot for help @torres-federico-e

**Comment by s-m-amin-ghasemi** [comment]
[1:50](https://youtu.be/pjq3QOxl9Ok?t=110) - Why Pandas? Introduction - Pandas vs. Python Builtins
[2:26](https://youtu.be/pjq3QOxl9Ok?t=146) - Loop+Sets Vs Pandas
[3:45](https://youtu.be/pjq3QOxl9Ok?t=225) - List and Dictionary Introduction
[4:26](https://youtu.be/pjq3QOxl9Ok?t=266) - Pandas Intersection of two Collections
[6:13](https://youtu.be/pjq3QOxl9Ok?t=373) - List Intersection of two Collections
[6:48](https://youtu.be/pjq3QOxl9Ok?t=408) - Dict Intersection of two Collections
[7:41](https://youtu.be/pjq3QOxl9Ok?t=461) - Counter Intersection of two Collections
[8:28 ](https://youtu.be/pjq3QOxl9Ok?t=508)- Pandas "+" Operation
[10:22](https://youtu.be/pjq3QOxl9Ok?t=622) - Pandas groupby
[11:29](https://youtu.be/pjq3QOxl9Ok?t=689) - Redefining variable in python will result as append pandas Dataframe
[12:50](https://youtu.be/pjq3QOxl9Ok?t=770) - Pandas "+" Operation and reset_index() key
[13:07](https://youtu.be/pjq3QOxl9Ok?t=787) - Pandas transform, apply, agg key
[17:14 ](https://youtu.be/pjq3QOxl9Ok?t=1034)- Pandas - Why so much complexity?
[18:41](https://youtu.be/pjq3QOxl9Ok?t=1121) - Numpy - Introduction
[26:44 ](https://youtu.be/pjq3QOxl9Ok?t=1604)- Numpy Conclusions
[27:15](https://youtu.be/pjq3QOxl9Ok?t=1635) - Xarray - Introduction
[31:44](https://youtu.be/pjq3QOxl9Ok?t=1904) - Xarray - Conclusions
[31:59](https://youtu.be/pjq3QOxl9Ok?t=1919) - Pandas - Series & Arrays
[32:50](https://youtu.be/pjq3QOxl9Ok?t=1970) - Pandas - Index introduction
[33:56](https://youtu.be/pjq3QOxl9Ok?t=2036) - Pandas - Conclusions and Problems
[34:47 ](https://youtu.be/pjq3QOxl9Ok?t=2087)- Series - Accessor ambiguity & Methods
[36:19](https://youtu.be/pjq3QOxl9Ok?t=2179) Why this Pandas ambiguity?
[39:10](https://youtu.be/pjq3QOxl9Ok?t=2350) Possible Errors & Indexes Types
[40:25](https://youtu.be/pjq3QOxl9Ok?t=2425) Index is a Mechanism - Demo
[41:46](https://youtu.be/pjq3QOxl9Ok?t=2506) Index translates: Human to Physical Memory
[45:11](https://youtu.be/pjq3QOxl9Ok?t=2711) Demo: Index Mechanics on Arithmetics
[47:07](https://youtu.be/pjq3QOxl9Ok?t=2827) Indexes are Operative metadata
[50:04](https://youtu.be/pjq3QOxl9Ok?t=3004) Back to Numpy - Rule comparisson
